src/components/card/Card.jsx

```diff
@@ -109,7 +77,36 @@ const Card = ({
     setDimensions(newStyle)
   }, [height])
 
-  const displayTwitterFooter = isTwitter && !mobileWebTwitter
+  const setCardSize = useCallback(() => {
```

Comment:
Any reason to have this inside the component as opposed to a standalone hook? IMO the latter is a lot cleaner since this contains so much nitty-gritty logic